image_rotate/src/image_rotate_node.cpp

```diff
@@ -48,8 +48,8 @@
 #include <vector>
 
 #include "cv_bridge/cv_bridge.hpp"
-#include "tf2/LinearMath/Vector3.h"
-#include "tf2/LinearMath/Quaternion.h"
+#include "tf2/LinearMath/Vector3.hpp"
+#include "tf2/LinearMath/Quaternion.hpp"
```

Comment:
we should really wrap this with an if/else so that the branch continues to compile on older systems (since I expect this change to tf2 will only be released into rolling and later at this point):

```cpp
#if __has_include(<tf2/convert.hpp>)
#include <tf2/convert.hpp>
#else
#include <tf2/convert.h>
#endif